git-hub

```diff
@@ -796,6 +797,37 @@ class CloneCmd (object):
 		git_quiet(1, 'fetch', remote)
 
 	@classmethod
+	def validate_url(cls, url, urltype):
+		scheme = urltype[:-4] # Remove the '_url'
+		# Any URL starting with <transport>:: will run
+		# a git-remote-<transport> command. "ext::" is specially
+		# unsafe, as that can run arbitrary commands.
+		if '::' in url:
+			pass # Fishy already, so skip to the question
+		elif scheme in ('clone', 'svn') and re.match(r'^https?://', url):
+			return
+		# XXX: git clone man page says it should be "git://",
+		# but github uses "git:"... I don't know how one is supposed to
+		# distinguish between an SSH URL without the "ssh://" and "git"
+		# as the host and this scheme... but well, accepting this as
+		# long as git knows how to deal with it should be safe.
+		elif scheme == 'git' and re.match(r'^git:', url):
+			return
+		elif scheme == 'ssh' and (re.match(r'^ssh://', url) or
+				re.match(r'^([^@:]+@)?[^:]+:', url)):
+			return
+		answer = ask("The URL reported by the GitHub server ({}) "
+			"looks a bit quite fishy! You might be under a "
```

Comment:
It was just a typo, originally I wrote `is a bit fishy' and then I intended to change it to `is quite fishy` :)